src/sentry/lang/javascript/processor.py

```diff
@@ -659,7 +658,16 @@ def __init__(self, organization, project=None, release=None, dist=None, allow_sc
         self.release = release
         self.dist = dist
         self.allow_scraping = allow_scraping
+        # Map that contains the mapping between bundle_id -> ArtifactBundleArchive to keep in memory all the open
+        # archives.
```

Comment:
```suggestion
        # Mappings between bundle_id -> ArtifactBundleArchive to keep all the open archives in memory.
```

Map contains mappings by definition :P